transport/src/test/java/io/netty/channel/embedded/EmbeddedChannelTest.java

```diff
@@ -605,6 +606,41 @@ public void channelInactive(ChannelHandlerContext ctx) throws Exception {
         assertTrue(inactive.get());
     }
 
+    @Test
+    public void testCloseAutoRead() {
```

Comment:
@carryxyh can you also add a test to ensure buffered messages are released when the `EmbeddedChannel` is closed but these are not processed ?

Reply:
Sure. 
Let me complete this.